.github/workflows/test-internal-prs.yml

```diff
@@ -111,7 +111,7 @@ jobs:
       repository: ${{ github.repository }}
 
   e2e:
-    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@main
+    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@liveness-main
```

Comment:
We should change to `@main` eventually before merge. This means that we have to test e2e workflow in some other PR, but looks like @0618  already opened one: #3678.